afs/templates/views/components/reports/instrument.htm

```diff
@@ -55,31 +55,57 @@ <h1 class="afs-report-toolbar-title"><span class="afs-report-name">{% trans "Ins
                         data: resource,
                         cards: documentationCards,
                         dataConfig: documentationDataConfig
-
-                    }
-                }"></div>
-            </div>
-            <!-- Existence Tab -->
-            <div class="afs-report-page" data-bind="if: activeSection() === 'existence'">
-                <div data-bind="component: {
-                    name: 'views/components/reports/scenes/existence',
-                    params: {
-                        data: resource,
-                        cards: existenceCards,
-                        dataConfig: existenceDataConfig
-                    }
-                }"></div>
-            </div>
-            <!-- Substance Tab -->
-            <div class="afs-report-page" data-bind="if: activeSection() === 'substance'">
-                <div data-bind="component: {
-                    name: 'views/components/reports/scenes/substance',
-                    params: {
-                        data: resource,
-                        cards: substanceCards
-
                     }
                 }"></div>
+                <div class="afs-report-section">
+                    <h2 class="afs-report-section-title"><i data-bind="click: function() {toggleVisibility(visible.textualReference)}, css: {'fa-angle-double-right': visible.textualReference(), 'fa-angle-double-up': !visible.textualReference()}" class="fa toggle"></i> {% trans "Textual Reference" %}</h2>
+                    <span data-bind="if: cards['textual reference to instrument'] && (!textualReference().length || cards['textual reference to instrument'].cardinality == 'n')">
```

Comment:
rather than repeating `cards['textual reference to instrument']` - just assign it to an observable on the viewmodel.  This is an example of DRY coding - don't repeat yourself if you don't have to.